src/open_inwoner/accounts/forms.py

```diff
@@ -59,6 +59,30 @@ class Meta:
         )
 
 
+class NecessaryUserForm(forms.ModelForm):
+    invite = forms.ModelChoiceField(
+        queryset=Invite.objects.all(),
```

Comment:
this will be evaluatied upon loading open-inwoner. Should be populated in the __init__

Reply:
I think you indeed misunderstood. Doing this means afaik executing the queryset when starting/reloading runserver, the queryset won't be executed when instantiating the form. In my experience you need to make your queryset a callable, in that case it should work as expected.

However it could be that my knowledge is rusty, I noticed that they have an example in the Django documentation which does the same: https://docs.djangoproject.com/en/4.0/topics/forms/modelforms/#a-full-example

Reply:
Lets leave it as is. I've tested adding a plantemplate on the testenvironment and this worked so I was incorrect, sorry!